boards/arm/mimxrt1050_evk/Kconfig.defconfig

```diff
@@ -71,8 +71,18 @@ config ETH_MCUX_0
 
 endif # NETWORKING
 
+config DISPLAY_DRIVER
+	string
```

Comment:
defconfigs should name a type, please remove `string` from both `DISPLAY_DRIVER`  and `CHARACTER_FRAMEBUFFER_SHELL_DRIVER_NAME`.

Reply:
ok